tests/contest/runtimetest/src/tests.rs

```diff
@@ -546,6 +546,27 @@ pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
     }
 }
 
+pub fn test_validate_root_readonly(spec: &Spec) {
+    let root = spec.root().as_ref().unwrap();
+    if root.readonly().unwrap() {
+        if let Err(e) = test_dir_write_access("/") {
+            let errno = Errno::from_raw(e.raw_os_error().unwrap());
+            if errno == Errno::EROFS {
+                /* This is expected */
+            } else {
+                eprintln!("readonly root filesystem, error in testing write access for path /");
+            }
+        }
+    } else if let Err(e) = test_dir_write_access("/") {
+        let errno = Errno::from_raw(e.raw_os_error().unwrap());
+        if errno == Errno::EROFS {
+            eprintln!("readt only root filesystem is false but write access for path / is err");
+        } else {
+            /* This is expected */
+        }
```

Comment:
I think for `readonly = false`, write access should not give any errors at all. So if it not ok, that should be considered error irrespective of errno.

Reply:
I fixed this.
https://github.com/youki-dev/youki/pull/2976/commits/c3c62fbf865aeaedfb6fdce1b33aba31301988ca